Tools/Scripts/cross-toolchain-helper

```diff
@@ -546,7 +546,7 @@ def maybe_quote_run_cmd(argv_list, cmd_key):
     return new_argv_list
 
 
-def main():
+def main(*args):
```

Comment:
I think this should be without asterisk.
If we use the asterisk here then we are passing to  `maybe_quote_run_cmd()` a tuple that contains inside as element zero the list instead of passing directly the list.